### PR TITLE
feat: create dnx txt record

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: src/commands/dns/txt-record/create.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      type: "string",
+      demandOption: true
+    })
+    .option("networkId", {
+      description: "Chain id to associate to the address",
+      type: "number",
+      demandOption: true
+    });
+
+const baseUrl = "https://612f62ma08.execute-api.ap-southeast-1.amazonaws.com/stg/";
```
Comment (Nebulis) [inline src/commands/dns/txt-record/create.ts:27]: wait for the real URL

Comment (Nebulis) [reply src/commands/dns/txt-record/create.ts]: done

File: src/commands/dns/txt-record.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Argv } from "yargs";
+import * as ethers from "ethers";
+ethers.errors.setLogLevel("error"); // disable warning from ethers
+
+export const command = "txt-record <method>";
+
+export const describe = "Invoke a function to interact with DNS TXT record";
```
Comment (rjchow) [inline src/commands/dns/txt-record.ts:7]: Methods to manipulate Issuer DNS-TXT records

**Pull request merged**